dropwizard-jdbi3/src/main/java/io/dropwizard/jdbi3/JdbiFactory.java

```diff
@@ -15,7 +16,7 @@
 import org.jdbi.v3.sqlobject.SqlObjectPlugin;
 
 public class JdbiFactory {
-    private final StatementNameStrategy nameStrategy;
+    protected final StatementNameStrategy nameStrategy;
```

Comment:
can this remain private?

Reply:
Yeah I could keep it private and send it as an argument to `buildSQLLogger`.